### PR TITLE
Add organizations webhooks

### PR DESCRIPTION
**Pull request by lucasmichot**
See https://developer.github.com/v3/orgs/hooks

### CODE REVIEW

File: lib/Github/Api/Organization/Hooks.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+     * @return array
+     */
+    public function all($organization)
+    {
+        return $this->get('orgs/'.rawurlencode($organization).'/hooks');
+    }
+
+    /**
+     * Get a single hook.
+     * @Link https://developer.github.com/v3/orgs/hooks/#get-single-hook
```
Comment (cursedcoder) [inline lib/Github/Api/Organization/Hooks.php]: link should be lowercase

**Comment by cursedcoder** [comment]
:+1: do you mind adding docs?

**Comment by lucasmichot** [comment]
Sure @cursedcoder

**Comment by lucasmichot** [comment]
Seems TravisCI is locked once more...

**Comment by cursedcoder** [comment]
@lucasmichot I will restart the build if needed.

**Comment by lucasmichot** [comment]
K

**Pull request merged**